NuRadioReco/modules/channelAntennaDedispersion.py

```diff
@@ -0,0 +1,77 @@
+from NuRadioReco.modules.base.module import register_run
+import numpy as np
+import copy
+import logging
+from NuRadioReco.utilities import units
+from scipy import signal
+from NuRadioReco.detector.antennapattern import AntennaPatternProvider
+from functools import lru_cache
+
+logger = logging.getLogger('channelAntennaDedispersion')
+
+
+class channelAntennaDedispersion:
+    """
+    
+    """
+
+    def __init__(self):
+
+        self._provider = AntennaPatternProvider()
+
+#     @lru_cache(maxsize=32)  # hashing doen't make sense because ff array is always different
+    def _get_response(self, det, station_id, channel_id, ff):
+        antenna_name = det.get_antenna_model(station_id, channel_id)
+        antenna = self._provider.load_antenna_pattern(antenna_name)
+        zen_ori, az_ori, zen_rot, az_rot = det.get_antenna_orientation(station_id, channel_id)
+        zen, az = None, None
+        if("LPDA" in antenna_name):
+            zen = zen_ori
+            az = az_ori
+        elif("bicone" in antenna_name or "dipole" in antenna_name):
+            zen = 90 * units.deg + zen_ori
```

Comment:
Is it documented somewhere that LPDA and bicone have their orientation defined differently?

Reply:
We should really make this consistently, or issue some warnings somewhere. I was not aware that this works like this.

Reply:
The orientation is not defined differently. The first vector is the orientation/direction of the antenna. But the sensitive direction for LPDAs and dipoles is different (by exactly 90 degrees). The LPDA is most sensitive when the signal comes from the boresight direction. A dipole is most sensitive when the signal comes in perpendicular to the antenna orientation (thus +90 deg + antenna orientation zenith angle).